lib/index.js

```diff
@@ -1,66 +1,48 @@
 var canvas = document.getElementById('game');
 var context = canvas.getContext('2d');
 
+// DINOSAUR stuff --------------------------------------------------------------
+var dino_img_right = document.getElementById("dino_right");
+var dino_img_left = document.getElementById("dino_left");
+dino_img_right.style.visibility = 'hidden';
+dino_img_left.style.visibility = 'hidden';
+
+var jumpSteps = 15;
```

Comment:
Q: perhaps we make this instance variables within dinosaur instead of leaving it out in the global namespace?